src/events.c

```diff
@@ -41,6 +45,35 @@ static void inform_keyboard_attached(gboolean value)
         }
 }
 
+static void dir_changed_cb(GFileMonitor *monitor,
+               GFile *file, GFile *other_file,
+               GFileMonitorEvent event_type, gpointer user_data) {
+    (void)monitor;
+    (void)other_file;
+    (void)user_data;
```

Comment:
Why is this needed?

Reply:
Often, C compiles will complain about unused arguments passed to a function. Putting `(void)` in front of the argument silences these warnings. So if would go for `-Wall -Wextra -Werror`, not having this code would cause problems.

Reply:
Then I don't understand why you didn't put all unused arguments here. I counted five.
If the compiler doesn't complain, I'd better delete this, but it's up to you.

Reply:
Because I copied that part of the code from MCE, which does use the other variables. Oh, how copy-paste introduces errors. :) Will fix.